src/shared/components/FormComponents/EditorController/EditorController.utils.ts

```diff
@@ -0,0 +1,12 @@
+const SPECIAL_SYMBOLS_LENGTH = 4;
+
+export const getValueLength = (value = '') => {
```

Comment:
In my opinion, this function needs to be improved to match not only one special case with "**", but also other different cases in the editor, where special characters are used.
For example:
![image](https://github.com/ChildMindInstitute/mindlogger-admin/assets/116339462/bfd45408-a4fd-48a1-8fdd-96e40530f0ba)